examples/3d_examples/MagneticCiliaCarpetCase/magnetic_cilia_carpet.py

```diff
@@ -26,10 +32,40 @@ class MagneticBeamSimulator(
 
         self.plot_result = plot_result
         self.magnetic_beam_sim = MagneticBeamSimulator()
-        # setting up test params
-        n_rods = num_rods_along_x * num_rods_along_y
+
+        # ====================== Setting up test params ====================
+        # Key parameters
         self.rod_base_length = rod_base_length
-        self.spacing_between_rods = self.rod_base_length  # following Gu2020
+        magnetic_field_strength = 80e-3
+        youngs_modulus = 1.85e5
+        density = 2.39e3
+        aspect_ratio = 5.0
+        shear_modulus = 6.16e4
+        spatial_magnetisation_phase_diff = np.pi  # Antiplectic
+
+        # Derived parameters
+        base_radius = self.rod_base_length / aspect_ratio / 2.0
+        base_area = np.pi * base_radius**2
+        volume = base_area * self.rod_base_length
+        moment_of_inertia = np.pi / 4 * base_radius**4
+        first_mode_frequency = (
+            3.5160
+            / rod_base_length**2
+            * np.sqrt(  # rad / s
+                youngs_modulus * moment_of_inertia / density / base_area
+            )
+        )
+
+        self.angular_frequency = frequency_ratio * first_mode_frequency
```

Comment:
a comment here for now, about this ratio, is would be helpful. we will later incorporate it into the documentation of the case.

Reply:
Fixed in [9d5f50f](https://github.com/SophT-Team/SophT/pull/120/commits/9d5f50f36230aa8060c2efc016e620444b432a39)